### DISCUSSION TITLE
Summer 2023

### DISCUSSION THREAD

**Discussion by Tolga1452** [discordexperimenthub/deh-assets#3] [Seasonal Theme]
### New Seasonal Theme!

We are looking some cool seasonal theme design(s) for our brand! We'll make a vote on 8 July and the designs will stay during 9 July-25 August, 2023.

- If you have any idea,
- If you are interested or have any design,
please share it in this discussion!

Do not forget, we'll add your name to our contributor list if you help us to make a design!

**Comment by Lordcobcob** [comment]
hello guys i can delete this if i want hehe

**Comment by Lordcobcob** [comment]
Currently working on a role icon for asset contributors :>

**Comment by Lordcobcob** [comment]
It'll come with a new role aswell around the same level as contributor

**Comment by hyuuuka** [comment]
![DEH_Summer_Logo](https://github.com/discordexperimenthub/deh-assets/assets/86443416/47a45e7f-18fa-4608-b6f3-516f329db04a)

**Reply by Tolga1452** [reply]
cool, thanks! we'll add it to our voting

**Comment by Zyrenth** [comment]
I was experimenting with these styles, here is a prototype:
@Tolga1452 a feedback would be helpful :3

![dehsummer-logo-1](https://github.com/discordexperimenthub/deh-assets/assets/71086259/7a11ee27-79dc-4da2-b64d-32f5756935a9)

**Reply by Tolga1452** [reply]
whoa, that's awesome! honestly i couldn't find anything missing on your design

**Reply by Zyrenth** [reply]
oh well tysm

**Reply by hyuuuka** [reply]
that's amazing :))

**Comment by AJtheCrocodile** [comment]
Hello, I’m interested in creating a design for the Discord Experiments Hub! Seeing the new changes being made to Discord all in one place is always neat, and I appreciate all the work that the team does. The best way that I can think of to give back to the community is by creating designs for the server 💖
I’m thinking of something simple that has a summer vibe and works with the already existing elements of the server icon while not changing it or covering it up, like filling the vial with a tropical drink or sticking it in the sand on a beach drawn in Discord’s simplistic art style. Doing something to “personify” the logo by giving it accessories associated with summer would be cute as well. I’ll be sure to return with concepts of these ideas!

**Reply by Tolga1452** [reply]
oh cool! thank you for your feedback and interest! we'll be waiting for your designs, but don't forget we'll make a voting with all submissions on 8 july (yeah i didn't include that will update the post and make a new announcement in DEH)

**Reply by AJtheCrocodile** [reply]
lol that’s alright, I’ll keep that in mind :] Thank you!!

**Comment by hyuuuka** [comment]
Here is another one, inspired by the summer colors palette on the internet and some people on the Discord Experiments Hub server !

![sasvavsavav](https://github.com/discordexperimenthub/deh-assets/assets/86443416/dc302647-e962-41c6-9931-feb0a26b4991)

**Reply by Tolga1452** [reply]
another cool one, thanks! i would like to see these waves on our main logo too 👀 but this is another discussion's topic

**Reply by hyuuuka** [reply]
alright :)

**Comment by AJtheCrocodile** [comment]
Hello again, here’s my logo submission for the summer design contest!! I tried doing something that only uses Discord’s usual promo art color palette :) Animated and still images attached 
![IMG_5899](https://github.com/discordexperimenthub/deh-assets/assets/138383996/ea08e8d3-acf4-49f1-9d4f-059bf97fc592)
![IMG_5900](https://github.com/discordexperimenthub/deh-assets/assets/138383996/8dfaf4d3-5a91-4403-baf6-5902f957093f)

**Reply by Bonkand** [reply]
in case you guys set this animated pfp you can use ezgif to make it loop once, so it stays as the potion

**Reply by Tolga1452** [reply]
yeah one animation for each hover would be better

**Reply by Tolga1452** [reply]
wait where's my first comment... sorry @AJtheCrocodile ig my network dead at this time. your work is so cool, thank you!

**Reply by AJtheCrocodile** [reply]
I set the first frame as the end frame too so it stays!
edit: You might’ve been talking about something different I just meant that the first frame of the icon would show the full design instead of the beginning of the animation

**Reply by Tolga1452** [reply]
@AJtheCrocodile you are winner of the vote! Can you please create a pull request (if possible, with `svg`, `png` and `gif` formats)?

**Reply by Tolga1452** [reply]
Also i would like to see the animation in regular logo too 👀

**Comment by AJtheCrocodile** [comment]
![Uploading IMG_5899.gif…]()


> Also i would like to see the animation in regular logo too 👀



> @AJtheCrocodile you are winner of the vote! Can you please create a pull request (if possible, with `svg`, `png` and `gif` formats)?

I tried to create an issue or pull request, hopefully it worked out :) My art program doesn’t support svg files, so I can only send png and gifs. I can create an animation for the *regular logo as well, it probably wouldn’t take any time over an hr to recolor and create a finished loop!
